src/lib/utils.ts

```diff
@@ -119,21 +16,26 @@ export async function setupApis(
  *  Ive spent hours trying to mock download
  *
  * @param apiFamily - Api family to download
- * @param deployment - What deployment to build for
  * @param rootPath - Root path to download to
+ *
  * @returns a promise that we will complete
  */
 export async function updateApis(
   apiFamily: string,
-  deployment: RegExp,
-  rootPath: string
+  rootPath: string,
+  isOas: boolean
 ): Promise<void> {
   try {
     const apis = await download.search(
-      `category:"CC API Family" = "${apiFamily}"`,
-      deployment
+      `"${apiFamily}" category:Visibility = "External" category:"SDK Type" = "Commerce"`,
+      undefined,
```

Comment:
[NIT] can we add a small comment on why we passed undefined as it's not immediately clear unless you understand the `search` method